dropwizard-request-logging/src/main/java/io/dropwizard/request/logging/async/AsyncAccessEventAppenderFactory.java

```diff
@@ -16,11 +16,6 @@
      */
     @Override
     public AsyncAppenderBase<IAccessEvent> build() {
-        return new AsyncAppenderBase<IAccessEvent>() {
-            @Override
-            protected void preprocess(IAccessEvent event) {
-                event.prepareForDeferredProcessing();
-            }
-        };
+        return new AsyncAppenderBase<IAccessEvent>();
```

Comment:
Definitely, thanks for mentioning. It would be nice to collapse this and `io.dropwizard.logging.async.AsyncLoggingEventAppenderFactory` into a single class parameterized on the type of event, since they're now almost identical. This would be a breaking change, though.

I took the liberty of updating `AsyncLoggingEventAppenderFactory`'s Javadoc too, since it also incorrectly links to `AsyncAppenderFactory`.